src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
+
+        Ball bigball = new Ball();
+
+        bigball.setColor(fff.toString());
+        bigball.setNumber(randomNumber);
+
+        return bigball;
+
+    }
+
+    public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+    }
```

Comment:
You are creating several instances of 'Ball' class. It's better to use a loop for this purpose. Please, refer to the checklist.